src/Tile_Data.md

```diff
@@ -74,19 +74,22 @@ Byte 2-3  Second Line
 etc.
 ```
 
-For each line, the first byte specifies the least significant bit of the
-color ID of each pixel, and the second byte specifies the most significant bit.
-In both bytes, bit 7 represents the leftmost pixel, and
-bit 0 the rightmost. For example: let's say you have \$57 \$36 (in
-this order in memory), which in binary are 01010111 and 00110110.
-To obtain the color ID for the leftmost pixel,
-you take bit 7 of both bytes: 0, and 0. Thus the index is binary 00 = 0. For
-the second pixel, repeat with bit 6: 1, and 0. Thus the index is binary 01 =
-1 (remember to flip the order of the bits!). If you repeat the
-operation you'll find that the IDs for the eight pixels are 0 1 2 3 0 3
-3 1.
+For each line, the first byte specifies the least significant bit of the color
+ID of each pixel, and the second byte specifies the most significant bit. In
+both bytes, bit 7 represents the leftmost pixel, and bit 0 the rightmost. For
+example, the sprite \$3C \$7E \$42 \$42 \$42 \$42 \$42 \$42 \$7E \$5E \$7E \$0A
```

Comment:
Should the raw hex data be put in a monospaced font, without prefixes? That's how it's most often presented.

Reply:
I added the monospace but kept the $. I see arguments either way on the $: with it, it’s consistent with the rest of the explanation below; without it, it can be pasted directly into the linked tool.